pyquil/pragmas.py

```diff
@@ -0,0 +1,53 @@
+##############################################################################
+# Copyright 2016-2018 Rigetti Computing
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+##############################################################################
+
+from typing import List, Iterable
+
+from pyquil.quilbase import AbstractInstruction, Pragma
+from pyquil.quil import Program
+
+
+class InitialRewiring(Pragma):
+    def __init__(self, rewiring: str):
+        self.rewiring = rewiring
+
+    def out(self) -> str:
+        return f'PRAGMA INITIAL_REWIRING "{self.rewiring}"'
+
+
+NAIVE_REWIRING = InitialRewiring("NAIVE")
```

Comment:
A thought - what about making this an enum to group them together?

```python
class InitialRewiring(Enum):
    Naive = Pragma("PRAGMA INITIAL_REWIRING \"NAIVE\"")
    ....

my_program = Program(
    pragmas.InitialRewiring.Naive,
    ...
)
```

Then the enum also becomes documentation of what the available options are.